src/components/steps/StepBillingData.tsx

```diff
@@ -15,6 +15,10 @@ const CustomTextField = styled(TextField)({
 });
 
 const mailPECRegexp = new RegExp('^[\\w-\\.]+@([\\w-]+\\.)+[\\w-]{2,4}$');
+const fiscalCodeRegexp = new RegExp(
+  '^[A-Za-z]{6}[0-9lmnpqrstuvLMNPQRSTUV]{2}[abcdehlmprstABCDEHLMPRST]{1}[0-9lmnpqrstuvLMNPQRSTUV]{2}[A-Za-z]{1}[0-9lmnpqrstuvLMNPQRSTUV]{3}[A-Za-z]{1}$'
+);
+const onlyNumbersRegexp = new RegExp('^[0-9]{11}$');
```

Comment:
You can us the OR operator ( "|" ) to write a single regexp that will validate the taxCode and vatNumber fields